importer/src/test/java/org/hiero/mirror/importer/migration/ContractLogIndexMigrationTest.java

```diff
@@ -0,0 +1,151 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package org.hiero.mirror.importer.migration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hiero.mirror.importer.migration.ContractLogIndexMigration.INTERVAL;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.hiero.mirror.common.domain.contract.ContractLog;
+import org.hiero.mirror.common.domain.transaction.RecordFile;
+import org.hiero.mirror.importer.EnabledIfV1;
+import org.hiero.mirror.importer.repository.ContractLogRepository;
+import org.hiero.mirror.importer.repository.RecordFileRepository;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@EnabledIfV1
+@RequiredArgsConstructor
+@Tag("migration")
+class ContractLogIndexMigrationTest extends AbstractAsyncJavaMigrationTest<ContractLogIndexMigration> {
+
+    @Getter
+    private final ContractLogIndexMigration migration;
+
+    private final RecordFileRepository recordFileRepository;
+    private final ContractLogRepository contractLogRepository;
+
+    @AfterEach
+    void temporaryTableCleanedUp() {
+        assertThat(tableExists("processed_record_file_temp")).isFalse();
+    }
+
+    @Test
+    void migrationOnEmptyDB() {
+        // given, when
+        migration.migrateAsync();
```

Comment:
please merge from `main`, then call `runMigration()` from parent class.

```suggestion
        runMigration();
```

also need to add `waitForCompletion()` before assertions.

Reply:
Done, thanks.